scripts/nodes/n0_test

```diff
@@ -34,7 +34,7 @@ rm /tmp/main.*.log
   --num-mining-threads=1 \
   --miner-public-keys=BC1YLg7Bk5sq9iNY17bAwoAYiChLYpmWEi6nY6q5gnA1UQV6xixHjfV \
   --block-producer-seed='essence camp ghost remove document vault ladder swim pupil index apart ring' \
-  --starter-deso-seed='road congress client market couple bid risk escape artwork rookie artwork food' \
+  --starter-deso-seed='trash lake sense uphold bag nasty style gesture pizza hurry issue escape' \
```

Comment:
Thanks for sharing your seed :) Let's revert cause people probably have this saved up.